hooks/pre_prompt.py

```diff
@@ -1,9 +1,9 @@
 import warnings
 
-from ccds.version import __version__ as version
+from ccds import __version__
 
 if __name__ == "__main__":
-    if version < "2.0.1":
+    if __version__ < "2.0.1":
```

Comment:
Is there a scenario where this would still break? You could potentially put this in a try-except.

Reply:
Yeah I think that's a good idea regardless.